### PR TITLE
Adding Anime Frenzy

### PR DESCRIPTION
**Pull request by nate-moo**
is my git history broken? if so im sorry and wil try to fix it. if not then great!

### CODE REVIEW

**Comment by nate-moo** [comment]
`431: Request Header Fields Too Large`

File: anime_downloader/config.py [hunk 1]
```diff
@@ -31,6 +31,10 @@
         'provider': 'twist.moe',
     },
     "siteconfig": {
+        'animefrenzy': {
+            "lang": "subbed"
+            #Lang Key can either be "subbed" or "dubbed"
```
Comment (Blatzar) [inline anime_downloader/config.py:36]: I heavily doubt comments in config works

File: anime_downloader/config.py [hunk 2]
```diff
@@ -31,6 +31,10 @@
         'provider': 'twist.moe',
     },
     "siteconfig": {
+        'animefrenzy': {
+            "lang": "subbed"
```
Comment (Blatzar) [inline anime_downloader/config.py:35]: 'version' is often used for languages, I'd prefer that for consistency

**Comment by nate-moo** [comment]
@Blatzar I removed the comment from config.py, and changed the key from lang to version

**Pull request merged**